ui/src/variables/components/VariableFormContext.tsx

```diff
@@ -73,7 +73,7 @@ class VariableFormContext extends PureComponent<Props> {
     return <VariableForm {...props} />
   }
 
-  private handleHideOverlay() {
+  private handleHideOverlay = () => {
```

Comment:
doesnt this lose props when it goes through the transpiler?

Reply:
nope 😸

Reply:
that's super weird, cause it's binding to a context that is defined.... when.... I gotta go rebuild my mental model.